packages/react/src/tooltip-content/TooltipContent.tsx

```diff
@@ -0,0 +1,65 @@
+import { useComposedRefs } from "@radix-ui/react-compose-refs";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import { forwardRef, useRef } from "react";
+
+import { AnimatePresence } from "../animate-presence";
+import { Box, type BoxProps } from "../box";
+import { Text } from "../text";
+import { theme } from "../theme";
+import { useTooltipContext } from "../tooltip-context";
+import { Transition } from "../transition";
+import * as styles from "./TooltipContent.css";
+
+type TooltipContentProps = BoxProps<typeof RadixTooltip.Content>;
+
+export const TooltipContent = forwardRef<HTMLDivElement, TooltipContentProps>(
+  (
+    { align = "center", children, className, sideOffset = 2, ...props },
+    outerRef,
+  ) => {
+    const innerRef = useRef<HTMLDivElement>(null);
+    const ref = useComposedRefs(innerRef, outerRef);
+
+    const { keepOpenOnActivation, open } = useTooltipContext("TooltipContent");
+
+    return (
+      <AnimatePresence>
+        {open && (
+          <RadixTooltip.Portal forceMount>
+            <Transition duration="sm" type="pop">
+              <Box asChild {...styles.content({}, className)} {...props}>
+                <RadixTooltip.Content
+                  align={align}
+                  onPointerDownOutside={
+                    keepOpenOnActivation
+                      ? (event: CustomEvent) => {
+                          if (
+                            innerRef.current?.contains(
+                              event.target as Node | null,
+                            )
+                          )
+                            event.preventDefault();
+                        }
+                      : undefined
+                  }
+                  ref={ref}
+                  sideOffset={sideOffset}
+                >
+                  <Text fontSize="sm">{children}</Text>
+
+                  <RadixTooltip.Arrow
+                    fill={theme.colors["neutral.900"]}
```

Comment:
should be same as tooltip bg
```suggestion
                    fill={theme.colors["bg.neutral.inverse"]}
```

Reply:
this needs to be addressed still

Reply:
Missed it vai.